### DISCUSSION TITLE
Html in notifications preview

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#49935] [Mobile] [answered]
GitHub Mobile beta 

1. Get a notification for
     https://github.com/vectordotdev/vector/pull/15371
2. See `<h1>`
    ![Screenshot_20230313-115912.png](https://user-images.githubusercontent.com/2119212/224757980-f736a41a-d3bf-413e-82f6-779086c6a32e.png)

Expected results no raw html

**Comment by aconsuegra** [comment, marked as answer]
Thanks for this report @jsoref! We have had this issue since version 1.0 of the app, apologies. I'll revive the issue where we are tracking this bug, hoping to get more attention to it.
